code/src/App.js

```diff
@@ -1,9 +1,50 @@
 import React from 'react'
+import { createStore, combineReducers } from '@reduxjs/toolkit'
+import { TodoList } from 'components/TodoList'
+import { Provider } from 'react-redux'
+
+import { todos } from 'reducers/todos.js'
+
+
+
+const reducer = combineReducers({  todos: todos.reducer })
+
+const persistedState = localStorage.getItem("reduxState")
+? JSON.parse(localStorage.getItem("reduxState"))
+ : {};
+
+ const store = createStore(
+   reducer, 
+   persistedState,
+   window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
+   );
+
+ store.subscribe(() => {
+   localStorage.setItem("reduxState", JSON.stringify(store.getState()));
+ });
```

Comment:
Remember that if you do another project that uses localstorage and use the same name "reduxState", your browser will get the information from this project and your app might not work. 

Make this name very specific when you store information in localstorage